src/main/java/org/broadinstitute/hellbender/engine/filters/FragmentLengthReadFilter.java

```diff
@@ -16,7 +16,7 @@
 
     @Argument(fullName = "maxFragmentLength",
             shortName = "maxFragmentLength",
-            doc = "Keep only read pairs with fragment length at most equal to the given value",
+            doc = "Keep only read pairs with fragment length at most equal to the given value.",
```

Comment:
Insert length in GATK4 is using the HTSJDK insert size definition (difference between 5' end of read and 5' end of mate). I changed the doc here to point out this.

Let me know if I should change also the name of the filter...